genesyscloud/tfexporter/resource_genesyscloud_tf_export.go

```diff
@@ -169,6 +170,13 @@ func ResourceTfExport() *schema.Resource {
 				Optional:    true,
 				ForceNew:    true,
 			},
+			"use_legacy_architect_flow_exporter": {
```

Comment:
Should this be an attribute? Or should this be a feature flag env variable?
If it's kept as an attribute and we eventually want to remove it, won't that impact state?

Reply:
Yes we agreed an attribute on the exporter is the way to go for now. In the future, we can warn the user that the default value will change to `false` and eventually that we're going to remove the field altogether